test/restricted.run

```diff
@@ -0,0 +1,4 @@
+1
+test/CEU.22.map
+test/CEU.22.ped
```

Comment:
Commit and push these plink files

Reply:
???? They are there...

Reply:
Ah my mistake, didn't realize they were in the repo already